index.rst

```diff
@@ -30,6 +30,14 @@ You can subscribe to our Google Calendar containing all of our events
 Or, you can simply view the calendar `here <https://calendar.google.com/calendar/embed?src=mqcgc13fqifh9ocb7l8novguik%40group.calendar.google.com&ctz=America/Los_Angeles>`__.
 
 
+Meet and Analyze Data (MAD)
+----------------------------
+Weekly meetup with the data science community at UC Davis in the :doc:`Bennett Conference room <Bennett-room>` on the Veterinary Medicine campus from 3-5 pm on Wednesdays (updated Spring quarter 2018).
+
+* Have questions? Come, bring your questions! There are no silly questions. Researching your own answers prior to showing up is not necessary.
+* Have skills? Come! This is a relaxed co-working space with a group of friendly, computationally-minded people who enjoy discussing a diversity of data analysis problems and methods.
+* Want to come work in silence and eavesdrop on other peoples' struggles to gain inspiration? Great!
```

Comment:
As much as I would like to say "eavesdrop" (and I probably suggested this) I think it is maybe more diplomatic to say "listen in on problem-solving sessions to gain inspiration"